walkman.el

```diff
@@ -89,7 +105,7 @@ KEEP-HEADERS is a bool to tell wether or not to keep headers"
     (goto-char (point-min))
     ;; clean up the buffer
     (save-excursion
-      (while (search-forward "
" (point-max) t)
+      (while (search-forward-regexp "\\
\\|\\\\[1m\\|\\\\[0m" (point-max) t)
```

Comment:
Without this change the regex while searching headers doesn't work (and it leaves Emacs hanging for longer).